dev.Dockerfile

```diff
@@ -0,0 +1,29 @@
+# syntax=docker/dockerfile:1
+
+# Comments are provided throughout this file to help you get started.
+# If you need more help, visit the Dockerfile reference guide at
+# https://docs.docker.com/engine/reference/builder/
+
+FROM node:21-alpine
+
+# This will be mounted against the root of the repo on the host system
+WORKDIR /usr/src/app
+
+ENV NODE_ENV development
+
+# Run the application as a non-root user.
+USER node
+
+# Expose the port that the application listens on.
+EXPOSE 8443
+
+CMD npm install && \
```

Comment:
This feels pretty dodge actually. This whole Docker "image" is just "do nothing and then run a command against the host filesystem, assuming it's mounted...